src/graphql/main/mutations.ts

```diff
@@ -72,6 +73,7 @@ const fields = {
   onChainAddressCreate: OnChainAddressCreateMutation,
   onChainAddressCurrent: OnChainAddressCurrentMutation,
   onChainPaymentSend: OnChainPaymentSendMutation,
+  onChainUsdPaymentSend: OnChainUsdPaymentSendMutation,
```

Comment:
you are missing shield update, this mutation should use `isAuthenticated` in graphql-main-server

Reply:
https://github.com/GaloyMoney/galoy/pull/2072#discussion_r1058591644